xds/src/test/java/io/grpc/xds/XdsNameResolverTest.java

```diff
@@ -496,19 +496,19 @@ public void resolving_matchingVirtualHostNotFound_matchingOverrideAuthority() {
   public void resolving_matchingVirtualHostNotFound_notMatchingOverrideAuthority() {
     Route route = Route.forAction(RouteMatch.withPathExactOnly(call1.getFullMethodNameForPath()),
         RouteAction.forCluster(
-            cluster1, Collections.<HashPolicy>emptyList(), TimeUnit.SECONDS.toNanos(15L), null),
-        ImmutableMap.<String, FilterConfig>of());
+            cluster1, Collections.emptyList(), TimeUnit.SECONDS.toNanos(15L), null),
+        ImmutableMap.of());
     VirtualHost virtualHost =
         VirtualHost.create("virtualhost", Collections.singletonList(AUTHORITY),
             Collections.singletonList(route),
-            ImmutableMap.<String, FilterConfig>of());
+            ImmutableMap.of());
 
     resolver = new XdsNameResolver(null, AUTHORITY, "random",
         serviceConfigParser, syncContext, scheduler,
         xdsClientPoolFactory, mockRandom, FilterRegistry.getDefaultRegistry(), null);
     resolver.start(mockListener);
     FakeXdsClient xdsClient = (FakeXdsClient) resolver.getXdsClient();
-    xdsClient.deliverLdsUpdate(0L, Arrays.asList(virtualHost));
+    xdsClient.deliverLdsUpdate(0L, Collections.singletonList(virtualHost));
```

Comment:
What complained about this?

Reply:
IntelliJ warns about using Arrays.asList with a single value or no value.  In this case we have something that takes multiple values and just happen to only be passing one so it is actually clearer the original way, so I'm changing it back.

Reply:
Yeah, Arrays.asList() is quite advantageous in tests as it means tests are more similar. In production code Collections.singletonList() is generally superior because it is immutable.